### DISCUSSION TITLE
Used in vs2022 will fail

### DISCUSSION THREAD

**Discussion by frf373** [community/community#16085] [Copilot Conversations]
第一种失效情况：打开第一个解决方案时，使用copilot是正常的。关闭解决方案，重新打开一个解决方案的时候，copilot会失效。
第二种失效情况：用着用着，copilot就不提示。
以上两种情况重启vs2022后都可以解决，但是每次都重启vs2022，很不便利。
The first failure case: when opening the first solution, it is normal to use copilot. When you close a solution and reopen a solution, copilot will fail.

The second failure case: when used, copilot will not prompt.

The above two situations can be solved after restarting vs2022, but restarting vs2022 every time is very inconvenient.

**Comment by punkmonday** [comment]
而且会导致gotoall功能失败
